ElectronClient/gui/NoteText.jsx

```diff
@@ -1181,10 +1186,12 @@ class NoteTextComponent extends React.Component {
 		}
 
 		if (command.name === 'focusElement' && command.target === 'noteBody') {
-			fn = () => {
-				if (!this.editor_) return;
+			const layout = this.props.visiblePanes[0];
```

Comment:
Please pay attention when naming variables. An editor or a viewer is not a "layout" so while the code might work it doesn't make sense.

In fact you don't need to think much to find a good name since the prop is called "visiblePanes", so you can just name your variable "visiblePane".

Reply:
Sure. I'll just change that.

Reply:
Done.